src/cpp/client/client_context.cc

```diff
@@ -75,6 +76,10 @@ void ClientContext::set_call(grpc_call* call,
   }
 }
 
+std::unique_ptr<const AuthContext> ClientContext::auth_context() const {
```

Comment:
If you change the prototype to return a `const AuthContext *`, you can just return `nullptr` here, right?